enum.c

```diff
@@ -1670,6 +1670,58 @@ enum_one(int argc, VALUE *argv, VALUE obj)
     return result;
 }
 
+DEFINE_ENUMFUNCS(many)
+{
+    if (RTEST(result)) {
+	if (memo->v1 == Qundef) {
+	    MEMO_V1_SET(memo, Qfalse);
+	}
+	else if (memo->v1 == Qfalse) {
+	    MEMO_V1_SET(memo, Qtrue);
+	    rb_iter_break();
+	}
+    }
+    return Qnil;
+}
+
+/*
+ *  call-seq:
+ *     enum.many? [{ |obj| block }]   -> true or false
+ *     enum.many?(pattern)            -> true or false
+ *
+ *  Passes each element of the collection to the given block. The method
+ *  returns <code>true</code> if the block returns <code>true</code>
+ *  more than once. If the block is not given, <code>many?</code>
+ *  will return <code>true</code> only if more than one of the
+ *  collection members is true.
```

Comment:
I think it should be "are truthy" instead of "is true".

Reply:
That's right, I've fixed it!

Reply:
You may also want to replace the "if the block returns true" above with "… truthy".